docs/getting_started/index.rst

```diff
@@ -1,5 +1,19 @@
-Installation
-============
+===============
```

Comment:
Why do we need this line over `Getting Started` section?

Reply:
We don't need this. Removed.